src/main/java/frc/robot/Autonomous.java

```diff
@@ -41,42 +36,54 @@
 
     private final PIDController positionPIDController;
 
+    private final CANSparkMax leftSide;
+    private final CANSparkMax rightSide;
+
     public Autonomous(Waiter waiter, Drive drivetrain, BallPath ballPath){
         this.waiter = waiter;
         this.drivetrain = drivetrain;
+        this.leftSide = drivetrain.getLeftSide();
+        this.rightSide = drivetrain.getRightSide();
         this.ballPath = ballPath;
         this.positionPIDController = new PIDController(kP, kI, kD);
         this.positionPIDController.setTolerance(DRIVE_DIST_TOLERANCE);
     }
 
-    double calcTicks(double encoderTicks){
-        // Calculates ticks per revolution(shaft rotation)
-        double gearRatio = 8;
-
-        double ticks = encoderTicks / gearRatio * wheelCircumference; // ticksPer cancels so it's just encoderTicks / gearRatio
+    public void turn(AHRS gyro, double degree) {
+        gyro.reset();
```

Comment:
Is this `.reset()` call really needed here? Last I checked, calling this method on the NavX meant that it would reset the reading angle to degrees so that the current physical direction would be 0. But, I think this is a non-blocking method call, meaning you call it and then some real wall clock time will pass before the reading is actually reset, and in the meantime your next lines of code are already executing.

Reply:
removed it, angles are relative to current angle now